nservicebus/upgrades/asb-6to7.md

```diff
@@ -59,4 +59,14 @@ snippet:topology-selection-upgrade-guide
 
 When the `EndpointOrientedTopology` is selected, it is also necessary to configure [publisher names](/nservicebus/azure-service-bus/publisher-names-configuration.md), in order to ensure that subscribers receive event messages:
 
-snippet:publisher_names_mapping_upgrade_guide
\ No newline at end of file
+snippet:publisher_names_mapping_upgrade_guide
+
+## Keep connection string safe
+
+In version 7 and above, to avoid sharing sensitive data contained into connection strings, it's possibile to enable `UseNamespaceNameInsteadOfConnectionString` feature: 
```

Comment:
I'm confused a little. Upgrade guide is intended for thins that have changed between old and new version and **must** be done in order to work properly. If by default, connection strings are used, that means during upgrade there's nothing  one must do in order for solution to work. If enabling safe connection strings is optional, it shouldn't be in the upgrade guide. Please correct me if I'm wrong with assuming that by default we're backwards compatible and one has to opt in to enable safe connection strings. Thank you.